spec-tests/src/test/java/org/candlepin/spec/owners/OwnerResourceSpecTest.java

```diff
@@ -317,6 +322,18 @@ void onlySuperAdminCanRefresh() throws ApiException {
         ApiClient readWriteClient = ApiClients.trustedUser(readWriteUser.getUsername());
         ApiClient adminClient = ApiClients.trustedUser(adminUser.getUsername());
 
+        ProductDTO product = Products.random();
+
+        if (CandlepinMode.isHosted()) {
```

Comment:
Why do names have to match? The point of display name annotation is to allow nice sentence-like name without the need to use such a construct an actual test name. If we are to duplicate the name we could just skip the @DisplayName annotation completely and use @DisplayNameGenerator instead to convert camel-case to spaces.

Reply:
Because otherwise when we have a failing test, I have to do a find-string-in-project/folder to find the offending test to debug and fix it, whereas if I can see the name, I can jump to the symbol. This is even more of a thing as our test output keeps getting worse and worse.